internal/bootstrap/bootstrap.go

```diff
@@ -272,6 +272,37 @@ func writeDotEnv(keys []api.ApiKeyResponse, config pgconn.Config, fsys afero.Fs)
 	return utils.WriteFile(".env", []byte(out), fsys)
 }
 
+func GetPostgresURLNonPooling(config pgconn.Config, fsys afero.Fs) string {
+	// Initialize default envs
+	initial := map[string]string{
+		POSTGRES_URL: utils.ToPostgresURL(config),
+	}
+
+	// Populate from .env.example if exists
+	envs, err := parseExampleEnv(fsys)
+	if err != nil {
+		errors.Errorf("failed to parse .env.example: %w", err)
+		return ""
+	}
+	for k := range envs {
+		switch k {
+		case POSTGRES_URL_NON_POOLING:
+			initial[k] = utils.ToPostgresURL(config)
+		default:
+			// Skip other keys
+			continue
+		}
+	}
+
+	// Check if POSTGRES_URL_NON_POOLING is present
+	if value, exists := initial[POSTGRES_URL_NON_POOLING]; exists {
+		return value
+	}
+
+	errors.New("error fetching POSTGRES_URL_NON_POOLING")
+	return ""
+}
+
```

Comment:
Let's remove these changes for part 1.

Reply:
I mean we don't need the entire `GetPostgresURLNonPooling` method.

In branches/get, we can serialise a map to env file like this.

```
	if utils.OutputFormat.Value != utils.OutputPretty {
		envs := map[string]string{
			"POSTGRES_URL": postgresConnectionString,
		}
		return utils.EncodeOutput(utils.OutputFormat.Value, os.Stdout, envs)
	}
```